### ISSUE TITLE
syncbatchnorm for 3d cnn

### ISSUE THREAD

**Issue by fengshi2017** [apache/incubator-mxnet#12145]
Hello, during training I3d with videos, I simply replace gluon.nn.BatchNorm with gluon.nn.SyncBatchNorm but an error as following is reported. Could anyone explain how to solve this problem?

error info: mxnet/3rdparty/tvm/nnvm/include/nnvm/tuple.h:438: Check failed: dim == static_cast<int>(ndim()) (4 vs. 5) dimension do not match target dimension 4 vs 5

**Comment by eric-haibin-lin** [comment]
@zhanghang1989

**Comment by zhanghang1989** [comment]
you could reshape the data to (b,c,h,w) and then reshape back.

**Comment by vdantu** [comment]
@mxnet-label-bot [Gluon,  Question]

**Comment by sandeep-krishnamurthy** [comment]
@fengshi2017 - Is your issue resolved?

**Comment by sandeep-krishnamurthy** [comment]
Resolving the issue based on the suggested solution. Please reopen if issue still persists.
